apps/explorer/src/components/HighlightedHash.tsx

```diff
@@ -16,28 +16,51 @@ const HighlightedHash = ({
   text,
   textToCopy,
   middleEllipsis = false,
-  fontSize = 14,
+  fontSize = 15,
   maxWidth = 'auto',
   className
-}: HighlightedHashProps) => (
-  <div style={{ fontSize, maxWidth, wordBreak: middleEllipsis ? 'initial' : 'break-all' }} className={className}>
-    {middleEllipsis ? <TextMiddleEllipsis text={text} /> : text}
-    {textToCopy && (
-      <ButtonWrapper>
-        <ClipboardButton textToCopy={textToCopy} />
-      </ButtonWrapper>
-    )}
-  </div>
-)
+}: HighlightedHashProps) => {
+  const [baseHash, group] = text.split(':')
+
+  return (
+    <div style={{ fontSize, maxWidth, wordBreak: middleEllipsis ? 'initial' : 'break-all' }} className={className}>
+      {middleEllipsis ? (
+        <>
+          <HighlightedPart>
+            <TextMiddleEllipsis text={baseHash} />
+          </HighlightedPart>
+          {group && <GroupPart>:{group}</GroupPart>}
+        </>
+      ) : (
+        <>
+          <HighlightedPart>{baseHash}</HighlightedPart>
+          {group && <GroupPart>:{group}</GroupPart>}
+        </>
+      )}
```

Comment:
All this can be simplified to:

```suggestion
      <HighlightedPart>{middleEllipsis ? <TextMiddleEllipsis text={baseHash} /> : baseHash}</HighlightedPart>
      {group && <GroupPart>:{group}</GroupPart>}
```

Reply:
I included this in my PR